core/includes/database/schema.inc

```diff
@@ -658,7 +658,13 @@ abstract class DatabaseSchema implements QueryPlaceholderInterface {
    */
   public function createTable($name, $table) {
     if ($this->tableExists($name)) {
-      throw new DatabaseSchemaObjectExistsException(t('Table @name already exists.', array('@name' => $name)));
+      if (db_select($name)->countQuery()->execute()->fetchField() == 0) {
+        $this->dropTable($name);
+        watchdog('database', 'Dropped empty, already existing table @name before recreating it.', array('@name' => $name));
+      }
+      else {
+        throw new DatabaseSchemaObjectExistsException(t('Table @name already exists and has data. Can\'t automatically recreate it.', array('@name' => $name)));
```

Comment:
```suggestion
        throw new DatabaseSchemaObjectExistsException(t("Table @name already exists and has data. Can't automatically recreate it.", array('@name' => $name)));
```

See https://api.backdropcms.org/php-standards#quotes

> 2. Translated strings where one can avoid escaping single quotes by enclosing the string in double quotes. One such string would be "He's a good person." It would be 'He\'s a good person.' with single quotes. Such escaping may not be handled properly by .pot file generators for text translation, and it's also somewhat awkward to read.

Reply:
You completely right @klonos, I just copied patch from D7 issue, shame on me.

Reply:
No shame @findlabnet 🙂 ...thanks for taking the time to contribute ❤️